LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWanNetworkSrvModule/Program.cs

```diff
@@ -32,4 +32,5 @@
 };
 
 var configuration = NetworkServerConfiguration.CreateFromEnvironmentVariables();
+configuration.RunningAsIoTEdgeModule = false;
```

Comment:
Is this change intentional?

Reply:
No it was done in a previous PR where I stemmed this branch from, I need to roll it back

Reply:
Removed good catch! Thanks!